### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by d3v1an7**
Add build notes (so I don't have to hunt this down next time) and fix docker version check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Pharod
+# Pharod
```
Comment (dackroyd) [inline README.md:1]: Space changed to space?

File: pharod-start [hunk 2]
```diff
@@ -14,7 +14,7 @@ fi
 
 sudo mkdir -p /etc/resolver
 
-if [[ $(docker version --format "{{.Server.KernelVersion}}") == *-moby ]]; then
+if [[ $(docker info --format "{{.OperatingSystem}}") == "Docker for Mac" ]]; then
```
Comment (dackroyd) [inline pharod-start:17]: Does this work with the older docker version too?

**Review by dackroyd — APPROVED** [review]

**Comment by dackroyd** [comment]
Maybe we define a `Makefile` to do the build rather than documenting steps?

Are we ready to take ownership of this to our private BB instead, and setup some semblance of consistency with our other repos for builds, and access to internal tools etc?

File: README.md (hunk 1 above)
Comment (d3v1an7) [reply README.md:1]: Yeah, Github wasn't rendering it as a H1, so just deleted and rewrote. UNICODEZ.

File: pharod-start (hunk 2 above)
Comment (d3v1an7) [reply pharod-start:17]: Not sure tbh! But should we care in this case?

**Comment by d3v1an7** [comment]
@dackroyd agree, but maybe for another PR. Would make the homebrew install much cleaner as well! (https://github.com/ffxblue/homebrew-custom/pull/3)

**Pull request merged**